### ISSUE TITLE
(URGENT) Infinite loop in db_cache.rb with relaton 0.5.10

### ISSUE THREAD

**Issue by ronaldtse** [relaton/relaton#65]
```
fetching ISO/IEC 6523...
bundler: failed to load command: metanorma (/Users/me/.rbenv/versions/2.6.5/bin/metanorma)
SystemStackError: stack level too deep
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/relaton-0.5.10/lib/relaton/db_cache.rb:133:in `match'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/relaton-0.5.10/lib/relaton/db_cache.rb:133:in `filename'
```

**Comment by ronaldtse** [comment]
This is after I removed `relaton/` local folder.

**Comment by andrew2net** [comment]
Solved in version 0.5.11
